doc/v2/documentation_data_format.rst

```diff
@@ -540,39 +540,60 @@ Detailed field description
+  :ref:`observable table <v2_observable_table>`.
 
   The :ref:`observable table <v2_observable_table>` allows marking some
-  parameters as measurement-specific (see below). Their values for a given
-  measurement are specified in this column. The values are separated by
-  semicolons. The order and number of values must match the order and number of
+  parameters as measurement-specific (see `observablePlaceholders` there).
+  Their values for a given measurement are specified in this column.
+  The values are separated by semicolons.
+  The order and number of values must match the order and number of
   placeholders in the ``observablePlaceholders`` field of the corresponding
-  observable in the observable table. The values may be
-  either numeric values or the IDs of parameters from the
+  observable in the observable table. I.e., if the observable table contains
+  no observable placeholders for the given observable, this field must be
+  empty.
+  The values may be either numeric values or the IDs of parameters from the
   :ref:`parameter table <v2_parameter_table>`.
 
   Different lines for the same ``observableId`` may specify different
-  parameters. This may be used to account for condition-specific or
-  batch-specific parameters. This will translate into an extended estimation
-  parameter vector.
+  values. This may be used to account for condition-specific or
+  batch-specific parameters.
 
-  If there are no placeholders used, this column may be omitted.
+  If none of the observables use any observable placeholders,
+  this column may be omitted.
 
 - ``noiseParameters`` [NUMERIC, STRING OR NULL, OPTIONAL]
 
-  The measurement standard deviation or empty if the corresponding sigma is a
-  model parameter.
+  Measurement-specific overrides for placeholder parameters in the
+  `noiseFormula` declared in the
+  :ref:`observable table <v2_observable_table>`.
+
+  The :ref:`observable table <v2_observable_table>` allows marking some
+  parameters as measurement-specific (see `noisePlaceholders` there).
+  Their values for a given measurement are specified in this column.
+  The values are separated by semicolons.
+  The order and number of values must match the order and number of
+  placeholders in the ``noisePlaceholders`` field of the corresponding
+  observable in the observable table. I.e., if the observable table contains
+  no noise placeholders for the given observable, this field must be
+  empty.
+  The values may be either numeric values or the IDs of parameters from the
+  :ref:`parameter table <v2_parameter_table>`.
+
+  Different lines for the same ``observableId`` may specify different
+  values. This may be used to account for condition-specific or
+  batch-specific parameters.
 
-  Numeric values or parameter names are allowed. Same rules apply as for
-  ``observableParameters`` in the previous point.
+  If none of the observables use any noise placeholders,
+  this column may be omitted.
```

Comment:
Fine as is but since measurements can be split across multiple tables, and not all tables need to have the same columns, then the column may be omitted in some tables if those tables do not contain relevant observables.